packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -29,8 +34,24 @@ class VmServiceWrapper implements VmService {
       log: log,
       disposeHandler: disposeHandler,
     );
+    _initSupportedProtocols();
+  }
+
+  Future<void> _initSupportedProtocols() async {
+    supportedProtocols = await getSupportedProtocols();
+    final ddsProtocol = supportedProtocols.protocols?.firstWhereOrNull(
+      (Protocol p) => p.protocolName?.caseInsensitiveEquals('DDS') ?? false,
+    );
+    _ddsSupported = ddsProtocol != null;
+    _supportedProtocolsInitialized.complete();
   }
 
+  late final ProtocolList supportedProtocols;
```

Comment:
This is never used outside of `_initSupportedProtocols`. Can we make it a local and remove this member?